components/threading/global_threads/thread_item/thread_item.tsx

```diff
@@ -78,7 +78,15 @@ function ThreadItem({
         }
     }, [channel, thread?.post.channel_id]);
 
-    const participantIds = useMemo(() => thread?.participants?.map(({id}) => id), [thread?.participants]);
+    const participantIds = useMemo(() => {
+        const ids = thread?.participants?.flatMap(({id}) => {
+            if (id === post.user_id) {
+                return [];
+            }
+            return id;
+        }).reverse();
+        return [post.user_id, ...ids];
```

Comment:
Should we include `post.user_id` by default or only if `post.user_id` is also found in the participants list?

Reply:
as per https://github.com/mattermost/mattermost-server/pull/18235 the root poster won't be included in the participants list. So in global threads list we do the inclusion manually.

In thread footer the requirement is that we do not show the root poster if the user has not replied to the thread.

Reply:
Got it thank you @koox00 🙂